package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Develop Barefoot nomad frontend",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "dev": "webpack-dev-server --mode development --hot --open --config webpack.config.babel.js",
+    "coveralls": "jest --coverage",
+    "test": "jest && jest --coverage",
+    "dev": "webpack-dev-server --mode development --hot --open --config webpack.config.js",
```

Comment:
Webpack is configured in ES6. Therefore you need `@babel/register` to transpile. Replace script dev command with `"dev": "webpack-dev-server --mode development --hot --open --config webpack.config.babel.js",`